dev/tidb-in-kubernetes/maintain/restart.md

```diff
@@ -0,0 +1,155 @@
+---
+title: 重启 TiDB 集群
+category: how-to
+---
+
+# 重启 TiDB 集群
+
+本文描述了如何重启 Kubernetes 集群上的 TiDB 集群，包括重启某个 Pod，重启某个组件的所有 Pod 和 重启 TiDB 集群的所有 Pod。
+
+## 重启某个 Pod
+
```

Comment:
It would be better if we add a brief intro to this section.
```suggestion

本小节介绍如何重启某个 Pod，包括以下四种重启场景：

- [强制重启某个 Pod](#强制重启某个-pod)
- [优雅重启 PD Pod](#优雅重启-pd-pod)
- [优雅重启 TiKV Pod](#优雅重启-tikv-pod)
- [优雅重启 TiDB Pod](#优雅重启-tidb-pod)

```